### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the link for whats an admin account

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7294/

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ description: "Learn how an admin can set a password expiration policy for your b
 
 ## Before you begin
 
-This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](Overview of the Microsoft 365 admin center](../admin-overview/admin-center-overview.md).
+This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](microsoft-365/admin/add-users/about-admin-roles).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](/microsoft-365/admin/add-users/about-admin-roles).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**